tests/check.sh

```diff
@@ -31,10 +42,11 @@ do
     fi
     if [[ ${HASSHELLCHECK} == 1 ]]; then
         if [[ $shebang =~ ^#!/.*/bash.* ]] || [[ $shebang =~ ^#!/.*/env\ bash ]]; then
-            shellcheck -x "$f"
+            /usr/bin/shellcheck -x "$f"
```

Comment:
Do we need this change?

Reply:
I'm concerned for non-travis workflows where shellcheck may not be at `/usr/bin`. Maybe download shellcheck to a different directory and set PATH in the travis config?

Reply:
Okey dokey; let me give that a try